<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTextEdit" name="login_textEdit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>190</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="login_Button">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>190</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your name</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Display Label</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="display_table_friends">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="display_table_posts">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>961</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="display_returnButton">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>530</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QPushButton" name="display_addFriendButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="display_suggestFriendsTable">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="display_postTextInput">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>250</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="display_postButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>240</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Post!</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_recommendedLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Recommended</string>
    </property>
   </widget>
   <widget class="QLabel" name="display_friendsLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>290</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>friends</string>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_ban">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>650</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Ban</string>
    </property>
   </widget>
   <widget class="QPushButton" name="admin_unbanButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>650</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Unban</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
